src/app/manage-projects/manage-single-project/manage-single-project.component.ts

```diff
@@ -66,12 +66,13 @@ export class ManageSingleProjectComponent implements OnInit, AfterViewInit {
                 } else {
                     displayBackendError(err);
                 }
-                SpinnerUtil.hideSpinner();
             });
+            SpinnerUtil.hideSpinner();
```

Comment:
This line doesn't make sense. subscription from line 57 to 69 represents asynchronous call
Think logically. This here means that you:

1. show spinner, 
2. synchronously start async call (finishes immediately), 
3. hide spinner. 

Practically, this spinner won't be even visible. If it fixes closing some orphan spinner from somewhere else, it's unintentional. Fix closing the right spinner on the place where it should be closed.